README.md

```diff
@@ -21,25 +18,32 @@
 
 Bagel API is a REST API, developed in [Python](https://www.python.org/) using [FastAPI](https://fastapi.tiangolo.com/) and [pydantic](https://docs.pydantic.dev/).
 
-[Getting started](#getting-started) |
-[Testing](#testing) |
-[License](#license)
+- [Quickstart](#quickstart)
+- [Local installation](#local-installation)
+- [Testing](#testing)
+- [License](#license)
 
-</div>
 
-## Getting Started
+## Quickstart
+The API is hosted at https://api.neurobagel.org/ and interfaces with Neurobagel's graph database. Queries of the graph can be run using the `/query` route.
```

Comment:
Could you add an example of a query route? e.g. sex="female" or something like that?